README.md

```diff
@@ -6,86 +6,107 @@
+`tp.leak`, features computation cannot depend on future data, thereby preventing
+ unwanted, hard-to-debug, and potentially costly future leakage.
+
+ - **Optimized for Temporal data** 🔥: Temporian's core computation is
+ implemented in C++ and optimized for temporal data. Temporian can be more than
+ 1,000x faster than non-temporal-specific libraries when operating on temporal
+ data.
+
+ - **ML library agnostic:** Temporian does not perform any machine learning
+ tasks. However, it can be used with any machine learning library, such as
+ PyTorch, Scikit-Learn, Jax, or TensorFlow.
+
+<!--
 - **Iterative and interactive development** 📊: Users can easily analyze
   temporal data and visualize results in real-time with iterative tools like
   notebooks. When prototyping, users can iteratively preprocess, analyze, and
   visualize temporal data in real-time with notebooks. In production, users
   can easily reuse, apply, and scale these implementations to larger datasets.
 
-- **Avoids future leakage** 😰: Future leakage occurs during model training
-  when a model is exposed to data from future events, which leaks information
-  that would otherwise not be available to the model and can result in
-  overfitting. Temporian operators do not create leakage by default. Users
-  can also use Temporian to programmatically detect whether specific signals
-  were exposed to future leakages.
-
 - **Flexible runtime** ☁️: Temporian programs can run seamlessly in-process in
   Python, on large datasets using [Apache Beam](https://beam.apache.org/).
-
-- **Highly optimized** 🔥: Temporian's core is implemented and optimized in
-  C++, so large amounts of data can be handled in-process. In some cases,
-  Temporian is 1000x faster than other libraries.
-
-> **Note**
-> Temporian's development is in alpha.
+-->
 
 ## QuickStart
 
 ### Installation
 
-Temporian is available on [PyPI](https://pypi.org/project/temporian/). Install it with pip:
+Install Temporian from [PyPI](https://pypi.org/project/temporian/) with `pip`:
 
 ```shell
-pip install temporian
+pip install temporian -U
 ```
 
 ### Minimal example
 
-The following example uses a dataset, `sales.csv`, which contains transactional data. Here is a preview of the data:
+Consider a record of individual sale logs that contain the `timestamp`, `store`, and `revenue` of individual sales.
 
 ```shell
-$ head sales.csv
-timestamp,store,price,count
-2022-01-01,CA,27.42,61.9
-2022-01-01,TX,98.55,18.02
-2022-01-02,CA,32.74,14.93
-2022-01-15,TX,48.69,83.99
+$ !cat sales.csv
+timestamp,store,revenue
+2023-12-04 21:21:05,STORE_31,5071
+2023-11-08 17:14:38,STORE_4,1571
+2023-11-29 21:44:46,STORE_49,6101
+2023-12-20 18:17:14,STORE_18,4499
+2023-12-15 10:55:09,STORE_2,6666
 ...
 ```
 
-The following code calculates the weekly sales for each store, visualizes the output with a plot, and exports the data to a CSV file.
+Our goal is to compute the sum of revenue for each store at 23:00 every weekday (excluding weekends).
```

Comment:
Nitpicking, would change "23:00" by "11 pm". Americans surprisingly don't use the 24 hour clock ;)

Reply:
Good point.
*Done